crates/ra_syntax/src/syntax_node.rs

```diff
@@ -6,15 +6,10 @@
 //! The *real* implementation is in the (language-agnostic) `rowan` crate, this
 //! module just wraps its API.
 
-use ra_parser::ParseError;
+pub(crate) use rowan::{GreenNode, GreenToken};
 use rowan::{GreenNodeBuilder, Language};
 
-use crate::{
-    syntax_error::{SyntaxError, SyntaxErrorKind},
-    Parse, SmolStr, SyntaxKind, TextUnit,
-};
-
-pub(crate) use rowan::{GreenNode, GreenToken};
```

Comment:
The "re-export" was intentionally after "imports", but I dont' have strong feelings about htis.

Reply:
Hmm, if I am not mistaken (need to recheck), this is what rustfmt suggested me todo and I couldn't resist